lib/renderer.js

```diff
@@ -272,6 +272,10 @@
      * Get SVG data for the given component.
      * 
      * @private
+     * @param {Component} component with component.layer or component.comp, or neither.
+     *      If neither is specified, this method gets the SVG data for the whole document.
+     * @param {?Layer} component.layer Layer to get the SVG data for. Exclusive of component.comp.
```

Comment:
Thanks for updating the documents! Isn't the annotation for this something like ```{{layer: ?Layer, comp: ?Comp}}``` or something like that? I don't think we actually use the closure compiler but should try and keep they syntax somewhat close. 
https://developers.google.com/closure/compiler/docs/js-for-compiler#types